### ISSUE TITLE
Calling .enable() on Android never seems to resolve

### ISSUE THREAD

**Issue by joewood** [rusel1989/react-native-bluetooth-serial#1]
Using React-native 0.29 on Android 4.4.2.
I'm seeing an exception on Android when attempting to enable Bluetooth. This doesn't result in an error from the promise. The exception is:
```
E/BluetoothAdapter( 2176): Bluetooth binder is null
E/RequestPermissionActivity( 2176): Error: there's a problem starting Bluetooth
```
The permissions are set-up in the manifest as follows:
```
    <uses-permission android:name="android.permission.BLUETOOTH" />
    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
```

**Comment by rusel1989** [comment]
Thanks for feedback, I am on it

**Comment by mhagmajer** [comment]
+1

**Comment by rusel1989** [comment]
Hey guys sorry for the slight delay i was on vacation ... i will try to resolve this ASAP

**Comment by vasun77** [comment]
+1

**Comment by rusel1989** [comment]
Fixed in 0.1.1 , you can also listen for bluetoothEnabled and bluetoothDisabled events and use disable() to turn off bluetooth
